modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -276,14 +276,14 @@ trait TargetOptics { this: Target.type =>
     sidereal.andThen(SiderealTarget.epoch)
 
   /** @group Optics */
-  val properMotion: Optional[Target, Option[ProperMotion]] =
-    siderealTracking.andThen(SiderealTracking.properMotion)
+  val properMotion: Optional[Target, ProperMotion] =
```

Comment:
@swalker2m I think we had some discussion about this?

Reply:
> Should there be a test for this?

I'm not sure what we would test. We were testing `Optional` laws already, which passed.

I don't remember a discussion about this. Having `properMotion: Optional[Target, ProperMotion]` seems consistent with `properMotionRA: Optional[Target, ProperMotion.RA]` and `properMotionDec: Optional[Target, ProperMotion.Dec]`.

Reply:
https://github.com/gemini-hlsw/lucuma-core/pull/412

Reply:
See also: `val radialVelocity: Optional[Target, Option[RadialVelocity]]`

Reply:
Oh ok, I didn't remember that PR. I thought this was a copy/paste error on my part when I redid these optics a while ago.

It seems to me then that if we want to set `sideralTarget.properMotion` to `None` via optics we should have `SiderealTarget.properMotion` be a `Lens[SiderealTarget, Option[ProperMotion]` and `Target.properMotion = sidereal.andThen(SiderealTarget.properMotion)`.

Unless there's an objection to this, I'll make that change then.

Reply:
I think the problem is that Target is an ADT and non sideral cannot take a proper motion, this laws fail